components/org.wso2.bfsi.consent.management.extensions/pom.xml

```diff
@@ -0,0 +1,187 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.wso2.bfsi.consent.management</groupId>
+        <artifactId>bfsi-consent-management</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>org.wso2.bfsi.consent.management.extensions</artifactId>
+    <name>WSO2 BFSI - Consent Extensions</name>
+    <description>WSO2 BFSI - Consent Extensions Module</description>
+    <url>http://maven.apache.org</url>
```

Comment:
```suggestion
    <url>https://wso2.com</url>
```

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/4/commits/98992be82ffa4483d8c6f51e4dce81ee11bb9d3b